meldingen_core/wfs.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABCMeta, abstractmethod
+from importlib import import_module
+from typing import AsyncIterator
+
+from meldingen_core.models import AssetType
+
+
+class BaseWfsProvider(metaclass=ABCMeta):
+    @abstractmethod
+    async def __call__(
+        self,
+        type_names: str,
+        count: int = 1000,
+        srs_name: str = "urn:ogc:def:crs:EPSG::4326",
```

Comment:
These defaults feel pretty arbitrary for a generic core function. Should we just make these nullable instead? WFS can also be consumed with a different srs_name, and with a different output_format for example.